### DISCUSSION TITLE
How to find next Kubernetes version that is coming?

### DISCUSSION THREAD

**Discussion by frachel** [Azure/AKS#2632] [Q&A]
Is there a place to see what the next planned version(s) of Kubernetes to be supported are? For example right now if I were to create a cluster (useast) it would be "1.21.2". Is 1.21.3 or 1.21.4 coming soon? If so where could I look to see that info - in this repo? somewhere else?

**Comment by tomkerkhove** [comment]
This document can help you with that: https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli
